dali/test/python/operator_1/test_numba_func.py

```diff
@@ -51,6 +52,16 @@ def set_all_values_to_255_sample(out0, in0):
     out0[:] = 255
 
 
+def set_all_values_to_255_sample_gpu(out0, in0):
+    tx, ty, tz = cuda.grid(3)
+    x_s, y_s, z_s = cuda.gridsize(3)
+
+    for z in range(tz, out0.shape[0], z_s):
+        for y in range(ty, out0.shape[1], y_s):
+            for x in range(tx, out0.shape[2], x_s):
+                out0[z][y][x] = 255
```

Comment:
Can we have a test where we store a flattened thread id or similar, so we're sure that our coordinate system works properly?

Reply:
There is a test that does that more or less - `change_ndim_gpu`